caasp-ingress-nginx-controller-image/_service

```diff
@@ -0,0 +1,10 @@
+<services>
+    <service name="replace_using_package_version" mode="buildtime">
+        <param name="file">caasp-ingress-nginx-controller-image.kiwi</param>
```

Comment:
`nitpic`: May be the name 'caasp-nginx-ingress-controller-image` is better. Just to reflect the package used: https://build.suse.de/request/show/222963

Reply:
the package naming is incorrect imho. there are two nginx based ingress controllers: 

https://git.k8s.io/ingress-nginx/README.md

this is the one we're intending to use. it is called ingress-nginx controller. thats why I called the image that way. 

There is also a 2nd nginx based controller:


https://www.nginx.com/products/nginx/kubernetes-ingress-controller

This one is called 'NGINX Ingress Controller'. Thats not the one we want to use. hence the renaming of the package to reflect its true origin. 

see https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/ for documentation and links to the origins. we would like to use the kubernetes maintained nginx based ingress controller, also known as 'ingress-nginx'.